webpack.config.js

```diff
@@ -14,12 +14,20 @@ module.exports = function(env, argv) {
             filename: 'demo.bundle.[hash].js',
           }
         : undefined, // Otherwise the CleanWebpackPlugin will wipe our build during devserver
-    node: {
-      fs: 'empty', // Webex SDK `fs` dependency does not exist in browser.
-    },
+    node: {},
     devtool: argv.mode === 'production' ? 'source-map' : 'inline-source-map',
     resolve: {
       extensions: ['.js', '.jsx'],
+      fallback: {
```

Comment:
Could we have comments around what these changes are for?

Reply:
Sure, added